commit 11aae87a51dcab8b7b0a651bf115c4bcd64ad4c8 (refs/changes/06/1406/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-02-02T12:52:15-08:00 (2 years, 8 months ago)
    
    TOOLS-1673 sdcadm getAgentImages not working due to missing name attribute

diff --git a/lib/steps/agent-services.js b/lib/steps/agent-services.js
index 38e8c70..ae69da9 100644
--- a/lib/steps/agent-services.js
+++ b/lib/steps/agent-services.js
@@ -97,7 +97,7 @@ function agentServicesEnsureCreated(arg, cb) {
     function getAgentImages(callback) {
         vasync.forEachPipeline({
             func: function (agent, next) {
-                var name = agent.name;
+                var name = agent.metadata.SERVICE_NAME;
                 var imgUUIDPath = util.format(
                     '/opt/smartdc/agents/lib/node_modules/%s/image_uuid',
                     name);
